SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -1,50 +1,66 @@
-# Get started: 2 - Get and build the GDK’s Unreal Engine Fork
+# Get started: 2 - Get and build the SpatialOS Unreal Engine fork
 
-As the GDK extends Unreal Engine's networking capabilities at its core, to use it you first need to build our SpatialOS fork of Unreal Engine 4 from source.
+The GDK for Unreal extends Unreal Engine's networking capabilities at its core. To use it you first need to build our SpatialOS fork of Unreal Engine from source.
 
-### Unreal Engine EULA
+### Step 1: Unreal Engine EULA
 
-In order to get access to our fork, you need to link your GitHub account to a verified Epic Games account, agree to the Unreal Engine End User License Agreement ([EULA](https://www.unrealengine.com/en-US/eula)) and accept the invite to join the [EpicGames organisation on Github](https://github.com/EpicGames). You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github).
+To get access to our fork, you need to link your GitHub account to a verified Epic Games account, and agree to the Unreal Engine End User License Agreement ([EULA](https://www.unrealengine.com/en-US/eula)). You cannot use the GDK without doing this first. To do this, see the [Unreal Engine documentation](https://www.unrealengine.com/en-US/ue4-on-github).
 
-### Getting the Unreal Engine fork source code and Unreal Linux cross-platform support
-1. In a terminal window, clone the [Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK) repository. (You may get a 404 from this link. See  the instructions above, under _Unreal Engine EULA_, on how to get access.) <br/>
-Check out the fork by running either:
-    * (HTTPS) `git clone https://github.com/improbableio/UnrealEngine.git -b 4.20-SpatialOSUnrealGDK`
-    * (SSH) `git clone git@github.com:improbableio/UnrealEngine.git -b 4.20-SpatialOSUnrealGDK`
-1. To build Unreal server-workers for SpatialOS deployments, you need to build targeting Linux. This requires cross-compilation of your SpatialOS project and the Unreal Engine fork.
+### Step 2: Getting the Unreal Engine fork source code and Unreal Linux cross-platform support
 
-    In Unreal's [Compiling for Linux](https://wiki.unrealengine.com/Compiling_For_Linux) documentation, in the **getting the toolchain** section, click v11 **clang 5.0.0-based** to download the archive **v11_clang-5.0.0-centos7.zip** containing the Linux cross-compilation toolchain, then unzip.
+> You should clone the Unreal Engine fork into your root directory to avoid file path length errors. <br/>For example: C:\GitHub\UnrealEngine
```

Comment:
```suggestion
> **TIP:** Clone the Unreal Engine fork into your root directory to avoid file path length errors. <br/>For example: C:\GitHub\UnrealEngine
```
Should the clone tip come after the clone step??!